docker-compose.yml

```diff
@@ -14,8 +25,5 @@ services:
     ports:
       - 3306:3306
     command: --default-authentication-plugin=mysql_native_password
-
-  app:
-    build: ./
-    ports:
-      - 8080:80
+    volumes:
+      - db-data:/var/lib/mysql:rw
```

Comment:
Why is this necessary exactly?

Reply:
This way data is persistent for the MySQL server.